src/containers/Stories/index.jsx

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import { Container } from "react-bootstrap";
+import SingleStoryCard from "../../components/SingleStoryCard";
+//import i18next from "i18next";
+import LoadingPage from "../../components/LoadingPage";
+
+const Stories = () => {
+  //const [currentLang, setCurrentLang] = React.useState(i18next.language);
+  //const [langRef, setLangRef] = React.useState("10221");
+  const [posts, setPosts] = React.useState([]);
+  const _URL =
+    "https://public-api.wordpress.com/wp/v2/sites/nobarriers650349093.wordpress.com/posts";
+  // const langREF = {
+  //   en: "10221",
+  //   ar: "38299",
+  //   tr: "64053"
+  // };
+
+  React.useEffect(() => {
+    const fetchPosts = async () => {
+      const data = await fetch(_URL);
+      const fetchedPosts = await data.json();
+      setPosts(fetchedPosts);
+      // console.log(currentLang)
+      // switch(currentLang){
+      //   case "en":
+      //     setLangRef("10221");
+      //     break;
+      //   case "ar":
+      //     setLangRef("38299");
+      //     break;
+      //   case "tr":
+      //     setLangRef("64053");
+      //     break;
+      //   default:
+      //     setLangRef("10221");
+      //     break;
+      // };
+
+      // console.log(langRef)
+      // const filteredPosts = fetchedPosts.map((post)=>{
+      //   let langTag = post.tags.toString();
+      //   console.log(post)
+      //   if (langTag === langRef){
+      //     return post
+      //   }else{
+      //     console.log("nope")
+      //   }
+      // })
+      //setPosts(filteredPosts)
```

Comment:
This commented part is to filter the blogs with the translation. 
I commented it until we operate the i18n.

Reply:
@khaldoon9kh we shouldn't merge any commented code if it's not used then u have to move to to another branch this branch should include only the work for stories